src/mumble/OpusCodec.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2005-2018 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_OPUSCODEC_H_
+#define MUMBLE_MUMBLE_OPUSCODEC_H_
+
+#include <opus.h>
+
+#include <QtCore/QLibrary>
+
+#ifndef Q_OS_WIN
+#define __cdecl
+#endif
+
+class OpusCodec {
```

Comment:
Please add a class comment

I shouldn't have to read git log/blame to know what this class is about

Reply:
Also `CELTCodec` doesn't have it.

I think that we should add a comment for both in a separate pull request.

Reply:
No. This PR introduces this class. This class should be documented. This PR documents what the class is for. This PR should document this class.

Reply:
e,g, The OpusCodec class handles loading the shared Opus library and is a wrapper for that shared librarys functions.

Reply:
```
The OpusCodec class loads Opus from a shared library and acts as a wrapper for its functions.
```

What do you think?

Reply:
We only seem to have 3 `@brief` comments, and none of those use the style "The X type".

So maybe just use "Loads Opus from a shared library and acts as a wrapper for its functions."

Reply:
I agree.